pmoves/scripts/p7-agent-interpreter-test.sh

```
@@ -0,0 +1,91 @@
+done
+
+# --- reachability probe ---
+PROBE_URL="http://${TTS_HOST}:${TTS_PORT}/"
+PROBE_CODE=$(curl -s -o /dev/null --max-time 5 -w '%{http_code}' "$PROBE_URL" 2>/dev/null || echo "ERR")
+if [[ ! "$PROBE_CODE" =~ ^[23] ]]; then
+    if [[ "$JSON_OUT" -eq 1 ]]; then
+        jq -nc --arg host "$TTS_HOST" --arg port "$TTS_PORT" --arg code "$PROBE_CODE" \
+            '{stage:"reachability", host:$host, port:$port, http_code:$code, result:"FAIL"}'
+    else
+        echo "❌ Reachability FAIL: ${PROBE_URL} → HTTP ${PROBE_CODE}" >&2
+        echo "   Check Tailscale (`tailscale status | grep 5090`) and Pinokio overlay." >&2
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)